source/wayland/display.c

```diff
@@ -526,6 +526,100 @@ static const struct wl_keyboard_listener wayland_keyboard_listener = {
     .repeat_info = wayland_keyboard_repeat_info,
 };
 
+static void wayland_touch_down(void *data, struct wl_touch *wl_touch,
```

Comment:
The indentation is a bit inconsistent with the rest of the file.

Could you please run clang-format on the lines you've added?